ckanext/canada/helpers.py

```diff
@@ -218,11 +227,13 @@ def canada_date_str_to_datetime(date_str: str) -> 'datetime.datetime':
         if not m:
             raise ValueError('Unable to parse %s as seconds.microseconds' %
                              time_tuple[5])
-        seconds = int(m.groupdict().get('seconds'))
+        # type_ignore_reason: incomplete typing
+        seconds = int(m.groupdict().get('seconds'))  # type: ignore
```

Comment:
```suggestion
        seconds = int(m.groupdict()['seconds'])
```
would this let you remove the `type: ignore`? You know that there will be a seconds group and its value will be a string.

Reply:
yes it does.